PKHeX/MainWindow/Main.cs

```diff
@@ -4304,7 +4315,21 @@ public static void Reset()
                 return slotSource == form || slotDestination == form; // form already updated?
             }
         }
+        public static bool IsLinkedSlot(int slot, int box) {
```

Comment:
I recommend moving this to `SaveFile` and `SAV7`.
For `SaveFile`:
```
public virtual bool IsLinkedSlot(int slot, int box) {
    return false;
}
```
And use the same logic for the override in `SAV7`.